labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -50,7 +63,7 @@ object CollectionExercise02 {
    * using a functional approach.
    */
   def groupAdultsPerAgeGroup(persons: Seq[Person]): Map[Int, Seq[Person]] = {
-    error("fix me")
+    persons filter (_.age >= 18) sortBy (_.name) groupBy (_.age / 10 * 10)
```

Comment:
Yes, it's necessary. I originally didn't do it and couldn't figure out what it didn't like with my results. It isn't grouping by age, but by age bracket: so, 20-29, 30-39, etc.